### DISCUSSION TITLE
Nfpm: Install path recently changed?

### DISCUSSION THREAD

**Discussion by andig** [goreleaser/goreleaser#2966] [Q&A] [answered]
A couple of users complained that our app no longer installs to /usr/local/bin but /usr/bin which breaks the path encoded in the service: https://github.com/evcc-io/evcc/discussions/2825#discussioncomment-2325849.

I‘m wondering if anything changed in nfpm that we should be aware of? Our nightly releaser script is in https://github.com/evcc-io/evcc/blob/master/.goreleaser-nightly.yml

Much appreciated!

**Comment by caarlos0** [comment, marked as answer]
yes, binaries from external packages should not be installed to `/usr/local/bin`, as its the right place for user-compiled binaries.

you revert it back by changing `nfpms.bindir`: https://goreleaser.com/customization/nfpm/?h=bindir

--- 

_EDIT: typos_

**Reply by andig** [reply]
Found it. Seems this is https://github.com/goreleaser/goreleaser/pull/2910 and you are trying to say that /usr/bin is where foreign binaries *should* be installed.

**Reply by caarlos0** [reply]
oh, sorry, yes, will fix the comment

also added a note to the release notes

---

_EDIT: you guessed it, typos_

**Reply by andig** [reply]
Thank you, quick turnaround is much appreciated and release notes makes perfect sense
